oz/redis_sessions/middleware.py

```diff
@@ -24,7 +24,17 @@ def _session_key(self):
 
             if not session_id:
                 session_id = oz.redis_sessions.random_hex(20)
-                self.set_secure_cookie("session_id", session_id.encode('utf-8'), httponly=True)
+                if "cookie_domain" in oz.settings:
```

Comment:
Ah thanks for checking that, I need to remember to check the source code. I'll fix it